tests/Doctrine/Tests/DBAL/Functional/Types/LanguageDoubleTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\DBAL\Functional\Types;
+
+use const LC_ALL;
+use function setlocale;
+
+class LanguageDoubleTest extends DoubleTest
```

Comment:
Please avoid extending tests if possible. Specifically in this case, if you want to test the same code paths using different locales, use a data provider.

Reply:
@morozov 

Let me look into it.